#   gGDT: Grip detected (1 or 0)
#   gWDT: Work detected (1 or 0)
#   gWDR: Work dropped (1 or 0)
#   gSGF: Smart grip failed (1 or 0)
#   gMGS: Current magnet strength (0 ~ 100 [%])

uint16 gGDT
uint16 gWDT
uint16 gWDR
uint16 gSGF
uint16 gMGS